build-system/server.js

```diff
@@ -561,6 +561,22 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get('/dist/rtv/*/v0/*.js', function(req, res, next) {
```

Comment:
I'd rather now. First simply removing `/dist` doesn't work, I didn't investigate into it. Second we will need `/dist` in the file path I think?  Third this `/dist` helps me to match other paths' style here.